### PR TITLE
#1697 - Show Job/Job Type Max Tries

### PR DESCRIPTION
**Pull request by JohnPTobe**
##### Checklist

- [x] `manage.py test` passes
- [x] tests are included
- [x] documentation is changed or added

### Affected app(s)
- job

### Description of change
Added max_tries field to rest api for job and job type.

### CODE REVIEW

File: scale/docs/rest/v6/job.rst [hunk 1]
```diff
@@ -630,6 +631,8 @@ Response: 200 OK
 | error              | JSON Object       | The error that is associated with the job.                                     |
 |                    |                   | (See :ref:`Error Details <rest_v6_error_details>`)                             |
 +--------------------+-------------------+--------------------------------------------------------------------------------+
+| max_tries          | Integer           | Number of times this a job of this type will be automatically retried.         |
```
Comment (mikenholt) [inline scale/docs/rest/v6/job.rst]: minor typo. I think you meant "Number of times this job will be automatically retried."

File: scale/docs/rest/v6/job_type.rst [hunk 2]
```diff
@@ -410,6 +411,8 @@ Response: 200 OK
 | max_scheduled            | Integer           | Maximum  number of jobs of this type that may be scheduled to run at the |
 |                          |                   | same time. May be null.                                                  |
 +--------------------------+-------------------+--------------------------------------------------------------------------+
+| max_tries                | Integer           | Number of times this a job of this type will be automatically retried.   |
```
Comment (mikenholt) [inline scale/docs/rest/v6/job_type.rst]: minor typo. I think you meant "Number of times a job of this type will be automatically retried."

**Comment by JohnPTobe** [comment]
Added fix for #1695 as well.

File: scale/docs/rest/v6/job.rst (hunk 1 above)
Comment (JohnPTobe) [reply scale/docs/rest/v6/job.rst]: Good eye. Fixed with better wording.

File: scale/docs/rest/v6/job_type.rst (hunk 2 above)
Comment (JohnPTobe) [reply scale/docs/rest/v6/job_type.rst]: Good eye. Fixed with better wording.

**Pull request merged**